tests/integration/mininet_tests.py

```diff
@@ -6755,6 +6755,76 @@ def test_switching(self):
             source_host, overridden_host, rewrite_host, overridden_host)
 
 
+class FaucetDscpRewriteTest(FaucetUntaggedTest):
```

Comment:
This is technically not a `FaucetDscpRewriteTest` test, it's a `FaucetDscpMatchTest` or `FaucetDscpMatchEthDstRewriteTest`

Reply:
Done!